.github/workflows/ci.yml

```diff
@@ -56,16 +56,16 @@ jobs:
       fail-fast: false
       matrix:
         target:
-          - focal
+          - jammy
         include:
-          - target: focal
-            image_variant: focal
+          - target: jammy
+            image_variant: jammy
             lib_postfix: '/x86_64-linux-gnu'
     env:
       HOME: /home/runner
     runs-on: ubuntu-22.04
     container:
-      image: ghcr.io/kiwix/kiwix-build_ci_${{matrix.image_variant}}:38
+      image: ghcr.io/kiwix/kiwix-build_ci_${{matrix.image_variant}}:latest
```

Comment:
Please used here a pined version. If I'm not mistaken, this should be `2024-11-06`

Reply:
There is no such tag for `https://ghcr.io/kiwix/kiwix-build_ci_focal`

Reply:
Oops. I confused this PR with kiwix/kiwix-build#771

Reply:
Done